### ISSUE TITLE
We should not move recipe line from .lando.yml 

### ISSUE THREAD

**Issue by hkirsman** [wunderio/lando-drupal#8]
Currently in readme we say that recipe is optional to make sure upstream does not change this eg to drupal11 at some point.

But actually it's also needed to have composer and other Drupal related commands working before you have installed this package. So if you clone any project where lando-drupal is used and you have minimal "name: somesite" .lando.yml, then your next task would be to run lando composer install, but that would not work as .lando.base.yml is not in your project yet and Lando does not see the recipe line so it does not have composer command.

In other words recommended minimal local .lando.yml with this project is:
   ```
   name: your-project-name
   recipe: drupal10
   ```

We should update the readme and remove the optional comment from the installation step.
